### PR TITLE
feat: Create RollOrStop

### PR DESCRIPTION
**Pull request by ctaaag**
Create RollOrStop plan

### CODE REVIEW

**Comment by ctaaag** [comment]
check please

File: RollOrStop [hunk 1]
```diff
@@ -0,0 +1,4 @@
+RollOrStop
+
+나는 롤 or 스톱 기능을 구현하겠습니다
+하하하i
```
Comment (ggyu777) [inline RollOrStop]: i 킹받네요

**Review by ggyu777 — CHANGES_REQUESTED** [review]
i 삭제 요망

**Review by ggyu777 — COMMENTED** [review]
굳쟙

**Comment by ctaaag** [comment]
ㄱㅅ

**Pull request merged**